frontend/src/pages/ProjectPage.tsx

```
@@ -42,33 +42,33 @@ function ProjectPage() {
   const { projectId } = useParams<{ projectId: string }>()
   const navigate = useNavigate()
   const isMobile = useIsMobile()
-  
+
   const { currentProject, fetchProject, isLoading: isLoadingProject } = useProjects()
   const { agents, fetchAgents } = useAgents()
   const { } = useFiles()
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)